.github/workflows/test_linux_cuda.yml

```diff
@@ -48,6 +48,8 @@ jobs:
       #  run: echo "BRANCH_NAME=$(echo $GITHUB_REF | awk -F'/' '{print $3}')" >> $GITHUB_ENV
 
       - uses: actions/checkout@v4
+        with:
```

Comment:
What's this?

Reply:
Not sure, I believe @ori-kron-wis made this change while debugging the tests. Could you explain Ori?

Reply:
yeah I wanted to see if the cuda test error we had was somehow related to the commit history depth, so I tried it in the branch. But eventually it was about cleaning local caches + updating packages on the linux machine.
we can revert this